graphql/user_resolver.go

```diff
@@ -6,45 +6,10 @@ import (
 
 	"github.com/evergreen-ci/evergreen/model/event"
 	"github.com/evergreen-ci/evergreen/model/patch"
-	"github.com/evergreen-ci/evergreen/model/user"
 	restModel "github.com/evergreen-ci/evergreen/rest/model"
 	"github.com/evergreen-ci/utility"
 )
 
-// ParsleyFilters is the resolver for the parsleyFilters field.
```

Comment:
Is this removal intended? Was the removal generated?

Reply:
now that we use the complete `APIDBUser` as the underlying model, these get automatically resolved so we don't have to write any resolver code for them!